notebooks/Testgrid_flakiness_detection.ipynb

```diff
@@ -0,0 +1,2907 @@
+{
```

Comment:
<p>I think you can make these example arrays easily readible, for example like this:</p><p> </p><pre class="ql-syntax" spellcheck="false">test_array = [1]*6 + [12]*11 + [1]*7
test_array_1 = [1]*6 + [12, 1]*8 + [1]*2
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Done